rts/Lua/LuaVFS.cpp

```diff
@@ -150,16 +151,21 @@ int LuaVFS::Include(lua_State* L, bool synced)
 	int loadCode = 0;
 	int luaError = 0;
 
-	if ((loadCode = LoadFileWithModes(fileName, fileData, GetModes(L, 3, synced))) != 1) {
+	const auto mode = GetModes(L, 3, synced);
+	if ((loadCode = LoadFileWithModes(fileName, fileData, mode)) != 1) {
+		std::string_view hint {""};
```

Comment:
Does this perform differently than:

```cpp
const std::string hint = loadCode == -1 ? "File not seen by VFS (missing or in different VFS mode)" : "";
```

Reply:
In theory a std::string would allocate some memory and copy that string (and then pass that copy to the printf) while string_view is just a pointer to the original. A compiler would probably be able to optimize it out though.